src/api/user-info.js

```diff
@@ -0,0 +1,94 @@
+import {AsgardeoSPAClient} from '@asgardeo/auth-react';
+import endpointConfig from '../configs/endpoint-config';
+
+const meEndpoint = `${endpointConfig.api.endpoints.me}`;
+
+const auth = AsgardeoSPAClient.getInstance();
```

Comment:
Shall we get the `httpRequest` directly rather than getting the auth. Also check if `2` space indentation is used in the entire app.

Reply:
We need to pass the request configuration as a parameter in the HTTP request. Given that we have various API calls with different request configurations, it makes sense to define this configuration in a reusable manner. WDYT?

Reply:
I was referring to something like this.

```
const httpRequest = AsgardeoSPAClient.getInstance().httpRequest.bind(AsgardeoSPAClient.getInstance());
```

The reason being, the `useAuthContext ` hook exposes a similar `httpRequest` hook that can be used to invoke API calls from withing components. It would make sense to keep the consistency here as well IMO.